### PR TITLE
Fix netlify build tutorial #1236

### PR DESCRIPTION
**Pull request by matsu4ki**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [x] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

#1236 

---

* [x] DO NOT include files inside `lib` directory.

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ server {
 
 1.  Login to your [Netlify](https://www.netlify.com/) account.
 2.  In the [dashboard](https://app.netlify.com/) page, click **New site from Git**.
-3.  Choose a repository where you store your docs, leave the **Build Command** area blank, fill in the Publish directory area with the directory of your `index.html`, for example it should be docs if you populated it at `docs/index.html`.
```
Comment (anikethsaha) [inline docs/deploy.md:81]: I am not sure whether `document file` is more approachable than `index.html` like the old one. 

Can you explain what problem you faced?

Comment (matsu4ki) [reply docs/deploy.md:81]: When I use `docs/index.html`, I caught build error.
I guess netlify changed that how to build.
I'm deploying on netlify for the first time.
So I don't know what netlify's old build method was.

貼り付けた画像_2020_06_23_1_25-2
貼り付けた画像_2020_06_23_1_25

Sample repository
https://github.com/matsu4ki/docsify-build-test-on-netlify

Comment (Koooooo-7) [reply docs/deploy.md:81]: I guess it seems ambiguous about the directory path description.
so, u filled the `Publish directory` with `docs` or another directory ?

Comment (matsu4ki) [reply docs/deploy.md:81]: under docs folder.

Comment (matsu4ki) [reply docs/deploy.md:81]: I see.
I was making a mistake about explanation sentence.

sorry🙇‍♂️

**Pull request closed without merging**